src/screens/ledger.tsx

```diff
@@ -60,11 +57,6 @@ export const LedgerScreen = () => {
         options={{
           title: 'Verify',
         }}
-        initialParams={{
```

Comment:
why did you remove these?

Reply:
![image](https://user-images.githubusercontent.com/69737893/206747876-efdcef64-1f7d-4a3a-9c74-90b561c51325.png)

Reply:
this screen is run for two different flows and this prop determines which screen is next.

Reply:
Ok I'll fix it, but then what should be `nextScreen ` in the ledger-accounts? It wasn't here before

Reply:
I try fix it